### DISCUSSION TITLE
I've created a port to Go

### DISCUSSION THREAD

**Discussion by DustinJSilk** [dinerojs/dinero.js#738] [Show and tell]
Hey!

I really enjoyed using Dinero.js, and wanted to use it across my Go server as well, so I've created a Go replica with the same API. (I couldnt find anything that was a drop in replacement).

I just wanted to share it here, and please let me know if you have any thoughts on it @sarahdayan 

https://github.com/DustinJSilk/dinero.go

**Comment by sarahdayan** [comment]
Hey @DustinJSilk, this is awesome, thanks for sharing! Really cool to see Dinero's API ported to another language.

Heads up that the project has had some changes since you built this (consolidated package structure, new formatting API, rounding options in `transformScale`, etc.). If you want to check out the latest and adapt the port, feel free!
